doc/data/messages/b/bad-mcs-classmethod-argument/details.rst

```diff
@@ -1 +1,8 @@
-You can help us make the doc better `by contributing <https://github.com/PyCQA/pylint/issues/5953>`_ !
+Used when a metaclass class method has a first argument named differently
+than the value specified in valid-metaclass-classmethod-first-arg option
+(default to `cls`), recommended to easily differentiate them from regular
+instance methods.
+
+Weird enough but usually naming of the first argument is `mcs` and pylint
+documentation says so but when someone specifies it then the lib throws
+C0204 error.
```

Comment:
I don't understand this comment. If something seems odd to you, then open an issue so we can discuss it. It makes no sense to add your personal dissent to the documentation.